### DISCUSSION TITLE
Suggestions for an autocomplete

### DISCUSSION THREAD

**Discussion by BrunoQuaresma** [shadcn-ui/ui#986] [General]
Hey folks, I want to build an autocomplete and my first try was to use the Command component but to make that happen I would have to make the CommandList "float" on top of the other elements below like the Google autocomplete does. Any suggestions? Thank you!

**Comment by exponent28** [comment]
Check out [issue #173](https://github.com/shadcn-ui/ui/issues/173#issuecomment-1594693306)
